disk-utils/mkswap.8.adoc

```diff
@@ -76,6 +79,9 @@ Specify the _ENDIANNESS_ to use, valid arguments are *native*, *little* or *big*
 *-o*, *--offset* _offset_::
 Specify the _offset_ to write the swap area to.
 
+*-s*, *--size* _size_::
+Specify the size of the created swap file, in bytes. This option only makes sense when used with *--file*.
+
```

Comment:
It would be probably better to change the current SYNOPSIS (and related man page information) from
```
 mkswap [options] device [size]
```
to
```
 mkswap [options] device [blocks]
```
It's confusing if we use `size` for two different things.

Reply:
Maybe you can also add complete SYNOPSIS line for the file
```
  mkswap [options] device [blocks]
  mkswap [options] --file filename --size size
```